app/helpers.php

```diff
@@ -28,9 +28,14 @@ function edit_image($image, $coords)
 function incrementTransactionId($request)
 {
     $transactionId = $request->header('X-Request-ID');
-    $transactionIdParts = explode('-', $transactionId);
-    $incrementedStep = $transactionIdParts[1] + 1;
-    $newTransactionId = $transactionIdParts[0] . '-' . $incrementedStep;
 
-    return $newTransactionId;
+    if ($transactionId) {
+        $transactionIdParts = explode('-', $transactionId);
+        $incrementedStep = $transactionIdParts[1] + 1;
+        $newTransactionId = $transactionIdParts[0] . '-' . $incrementedStep;
+
+        return $newTransactionId;
+    }
+
+    return null;
```

Comment:
should this return null if there isn't a transaction id? or should it create one?